skimage/measure/_moments.py

```diff
@@ -445,4 +445,11 @@ def inertia_tensor_eigvals(image, mu=None, T=None):
     if T is None:
         T = inertia_tensor(image, mu)
     eigvals = np.linalg.eigvalsh(T)
+    """
+    Floating point precision problems could make a positive
+    semidefinite matrix have an eigenvalue that is very slightly
+    negative.  This can cause problems down the line, so set values
+    very near zero to zero.
+    """
```

Comment:
```suggestion
    # Floating point precision problems could make a positive
    # semidefinite matrix have an eigenvalue that is very slightly
    # negative. This can cause problems down the line, so set values
    # very near zero to zero.
```